src/test/fuzz/tx_pool.cpp

```diff
@@ -230,7 +230,8 @@ FUZZ_TARGET_INIT(tx_pool_standard, initialize_tx_pool)
         Assert(it->second.m_result_type == MempoolAcceptResult::ResultType::VALID ||
                it->second.m_result_type == MempoolAcceptResult::ResultType::INVALID);
 
-        const auto res = WITH_LOCK(::cs_main, return AcceptToMemoryPool(chainstate, tx_pool, tx, bypass_limits));
+        CChainState& chainstate{node.chainman->ActiveChainstate()};
```

Comment:
I think this line could simply be removed, as it is merely shadowing another reference with the same name and content on top of the fuzz test?

Reply:
Good catch. Line removed in 4ec8b96.

Reply:
not addressed?

Reply:
Yes, it was addressed in 4ec8b96.

Reply:
In the future, it might be good to keep a clean history and not introduce unused stuff in one commit and remove it in the next.